en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md

```diff
@@ -314,6 +310,12 @@ Because this is a public method called on a `ModelAdmin`, which will often be ex
 in a back-end context using admin themes, it pays to ensure we're loading the front-end
 themes whilst rendering out the preview content.
 
+> [!TIP]
+> If the `ModelAdmin` you want to do this on is in some vendor module, you can apply
+> this action in an extension as well! Just remember to use the public methods where
+> protected properties are used below (e.g. `$this->urlParams['ID']` would become
+> `$this->owner->getUrlParams()['ID']`).
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.